system/jlm/playlist.xml

```diff
@@ -42,6 +42,13 @@
 	<!-- Temporarily excluding from z/OS due to : jdk11-zos/issues/567 -->
 	<test>
 		<testCaseName>TestJlmRemoteClassAuth</testCaseName>
+		<disables>
+			<disable>
+				<comment>Disabled on aix for https://github.com/eclipse-openj9/openj9/issues/11557</comment>
```

Comment:
Could you remove the text `Disabled on aix for` in the comment? We should only keep URL portion as this will be parsed via another script for running disabled tests.

Reply:
Done!